client/src/javascript/components/modals/settings-modal/DiskUsageTab.js

```diff
@@ -0,0 +1,134 @@
+import {Checkbox, Form, FormRow, Select, SelectItem, Radio} from 'flood-ui-kit';
+import {FormattedMessage, injectIntl} from 'react-intl';
+import React from 'react';
+
+import ModalFormSectionHeader from '../ModalFormSectionHeader';
+import SettingsStore from '../../../stores/SettingsStore';
+import SettingsTab from './SettingsTab';
+import SortableList from '../../general/SortableList';
+import DiskUsageStore from '../../../stores/DiskUsageStore';
+
+class DiskUsageTab extends SettingsTab {
+  tooltipRef = null;
+
+  state = {
+    diskItems: [],
+  };
+
+  componentWillMount() {
+    // assemble disk items from saved "mountPoints" and list of disks "disks"
+    const mountPoints = SettingsStore.getFloodSettings('mountPoints');
+    const disks = DiskUsageStore.getDiskUsage().reduce((a, c) => {
+      a[c.target] = c;
+      return a;
+    }, {});
+    const diskItems = [];
```

Comment:
I think think this could be simplified with a single assignment, for example
```js
const diskItems = DiskUsageStore.getDiskUsage().map(d => ({ 
  id: d.target, 
  visible: mountPoints.includes(d.target)
}))
```

Reply:
The selected items have to in the order that they appear in `mountPoints` for the reorder functionality to work. That's why I am doing it in this convoluted way.

Reply:
The `SortableList` component expects the `visible` key. I looked into changing it, but the User Interface Tab and the `torrentDetails` field in the database use `visible` already.